docs/src/main/paradox/hbase.md

```diff
@@ -0,0 +1,48 @@
+#HBbase connector
+
+A flow and a composite sink to write element in [HBase](http://hbase.apache.org).
+
+HBase is a column family NoSQL Database backed by HDFS.
+
+ 
+#Usage
+
+Build a converter and a tableSetting.
+
+Converter will map the domain object to HBase column.
+
+Table will be created on demand.
+
+##scala
+
+@@snip (../../../../hbase/src/test/scala/akka/stream/alpakka/hbase/HBaseStageTestSuite.scala) { #create-converter }
+
+@@snip (../../../../hbase/src/test/scala/akka/stream/alpakka/hbase/HBaseStageTestSuite.scala) { #create-settings }
+
+###Flow usage 
+
+@@snip (../../../../hbase/src/test/scala/akka/stream/alpakka/hbase/HBaseStageTestSuite.scala) { #flow }
+
+###Sink usage
+
+@@snip (../../../../hbase/src/test/scala/akka/stream/alpakka/hbase/HBaseStageTestSuite.scala) { #sink }
+
+##java TBD
```

Comment:
I have started to write java stuff, but now that converter is a function, how can I convert a java function to a scala one ?